web-server/v0.4/.editorconfig

```diff
@@ -0,0 +1,16 @@
+# http://editorconfig.org
```

Comment:
Do we need this file?

Reply:
This is not needed and will be removed under this PR.